remix-lib/src/execution/txExecution.js

```diff
@@ -72,22 +72,23 @@ module.exports = {
     if (!txResult.result.vm.exceptionError) {
       return ret
     }
-    var error = `VM error: ${txResult.result.vm.exceptionError}.\n`
+    var exceptionError = txResult.result.vm.exceptionError.error || ''
```

Comment:
I think this cannot be properly merged here because it depends on #758, but `remix-ide` already had that changed merged so that it is broken without this change.

I wonder why is the VM a dependency of both modules?

Reply:
there's a pr to remove the ethereumjs-vm dep from remix-ide.. not yet merged